README.md

```diff
@@ -15,6 +15,10 @@ React.js
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
+<h2>Live Demo</h2>
+
+<a href="#" rel="noopener noreferrer">Live Demo Link</a>
+
 ## Available Scripts
 
 In the project directory, you can run:
```

Comment:
- [ ] I would recommend you do away with the dummy **`create-react-app`** texts that do not necessarily relate to the project in any way. I'd suggest you format your README using the Microverse template [here](https://github.com/microverseinc/readme-template)
- [ ] In your README remember to add instructions on how users can get your app repo locally, and also to run `npm install` to install packages and `npm start` to start project.

Reply:
Hi @TedLivist  and thank you for your suggestions 🙏 ! I have gone ahead and implemented those changes.